partner_firstname/tests/test_name.py

```diff
@@ -25,6 +25,12 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
 
+"""Test naming logic.
+
+Please have in mind that some tests will fail if the ``mail`` module is
```

Comment:
Yeah, I think so. It's not the best option, but if you don't know how to run the test with the other module installed, it can serve